app/models/project.rb

```diff
@@ -70,6 +70,12 @@ def is_not_final?
     self.title.blank? || self.language.blank? || self.description.blank? || self.github_link.blank?
   end
 
+  def tasks_completed_for_week?(week_number)
+    tasks_for_week = self.tasks.where(week: week_number)
+    return true if tasks_for_week.count >= 1 && tasks_for_week.unfinished_tasks_for_week(week_number, self.mentee)
```

Comment:
instead of those 2 returns you can simply return `tasks_for_week.count > 0 && tasks_for_week.unfinished_tasks_for_week(week_number, mentee).any?`

Reply:
sure.